packages/search-field/src/SearchField.js

```diff
@@ -44,29 +53,70 @@ class SearchField extends React.Component {
     this.debouncedOnChange.cancel(); // Avoid a debounced event to trigger after the component is unmounted
   };
 
-  onKeyUp = ({ key }) => {
-    if (key === "Enter") {
-      this.debouncedOnChange.flush();
+  onKeyDown = event => {
+    const { key } = event;
+    switch (key) {
+      case "Enter":
+        this.debouncedOnChange.flush();
+        break;
+      case "ArrowUp":
+        event.preventDefault();
+        this.setState(prevState => ({
+          highlightedIndex: Math.max(prevState.highlightedIndex - 1, -1)
+        }));
+        break;
+      case "ArrowDown":
+        event.preventDefault();
+        this.setState(prevState => ({
+          highlightedIndex: Math.min(
+            prevState.highlightedIndex + 1,
+            this.props.items.length - 1
+          )
+        }));
+        break;
```

Comment:
this feel redundant, but might be ok for v1, lets file an enhancement issue to track how to make either input-select or dropdown parts/behaviors more reusable